frontend/src/components/NotificationsBar/Notification/Notification.tsx

```diff
@@ -1,29 +1,37 @@
-import React from 'react';
+import { FC } from 'react';
 
-import { NOTIFICATION_STATUSES } from 'constants/statuses';
+import { NOTIFICATION_STATUSES, NOTIFICATION_TYPES } from 'constants/statuses';
 import { warningIcon, redWarningIcon } from 'icons';
-import { Notification } from 'types/notification';
+import { INotification } from 'types/INotification';
 
 import { NotificationContainer, ImageWrapper, Title, Description } from './styled';
 
 interface NotificationProps {
-  note: Notification;
+  note: INotification;
 }
```

Comment:
please move all the interfaces to separate file types.ts inside NotificationsBar folder

Reply:
and also they should be renamed from Prips/IProps to INotificationBarContainerProps and so on